packages/myst-cli-utils/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "myst-cli-utils",
+  "version": "0.0.1",
+  "license": "MIT",
+  "description": "Utils for logging, error messages and session management",
+  "author": "Rowan Cockett <rowan@curvenote.com>",
+  "homepage": "https://github.com/curvenote/curvenote#readme",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
```

Comment:
We had to take this block out of `myst-common` to get builds working

Reply:
Will leave it in and note it for later?